app/src/main/java/com/cornellappdev/android/eatery/ui/components/general/FilterRow.kt

```diff
@@ -218,6 +217,61 @@ fun CompareFilterRow(
     }
 }
 
+@Composable
+fun ItemFavoriteFilterRow(
+    currentFiltersSelected: List<Filter>,
+    onFilterClicked: (Filter) -> Unit
+){
+    LazyRow(horizontalArrangement = Arrangement.spacedBy(8.dp)) {
+        item {
+            Spacer(modifier = Modifier.width(10.dp))
+        }
```

Comment:
The way this whole file is structured is so sus. We should really try to make these `items` instead of `item` if possible. Zach's PR touches on this, but that needs to be merged first before we tackle this.

Plz add a TODO to this (and the other sus filter row composables here) to make them more concise :-)

Reply:
done!